server/server.go

```diff
@@ -24,9 +24,10 @@ const file_suffix string = ".cwf"
 
 var filesDir string
 var port int
+var config entities.ServerConfig_t
```

Comment:
If like this then is it not possible to just create it here 
var config =  entities.ServerToml_t or something like that